testsuite/integration-tests/pom.xml

```diff
@@ -178,6 +183,160 @@
+                    </plugin>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-jar-maven-plugin</artifactId>
+                        <executions>
+                            <!-- Package a jaxrs-server layer with a few other layers -->
+                            <execution>
+                                <id>bootable-jar-observability-packaging</id>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                                <phase>process-test-resources</phase>
+                                <configuration>
+                                    <output-file-name>jaxrs-server.jar</output-file-name>
+                                    <hollowJar>true</hollowJar>
+                                    <record-state>false</record-state>
+                                    <log-time>true</log-time>
+                                    <plugin-options>
+                                        <jboss-fork-embedded>true</jboss-fork-embedded>
+                                    </plugin-options>
+                                    <feature-packs>
+                                        <feature-pack>
+                                            <groupId>${testsuite.wildfly.galleon.pack.group.id}</groupId>
+                                            <artifactId>wildfly-galleon-pack</artifactId>
+                                            <version>${server.version}</version>
+                                            <!-- Required for rxjava -->
+                                            <includedPackages>
+                                                <package>org.jboss.resteasy.resteasy-rxjava2</package>
+                                            </includedPackages>
+                                        </feature-pack>
+                                    </feature-packs>
+                                    <layers>
+                                        <layer>jaxrs-server</layer>
+                                        <layer>microprofile-config</layer>
+                                        <layer>datasources</layer>
+                                        <layer>h2-default-datasource</layer>
+                                        <layer>undertow-https</layer>
+                                        <layer>ejb</layer>
+                                    </layers>
+                                    <channels>
+                                        <channel>
+                                            <manifest>
+                                                <groupId>${channel-manifest.groupId}</groupId>
+                                                <artifactId>${channel-manifest.artifactId}</artifactId>
+                                                <version>${channel-manifest.version}</version>
```

Comment:
Is there any reason not to use the properties already defined?

```xml
<groupId>${wildfly.channel.manifest.groupId}</groupId>
<artifactId>${wildfly.channel.manifest.artifactId}</artifactId>
<version>${server.version}</version>
```

Reply:
@jamezp when testing the product we would have e.g.

```
	upstream:
		wildfly.channel.manifest.groupId: org.wildfly.channels
		wildfly.channel.manifest.artifactId: wildfly-ee
		server.version: 32.0.1.Final

		server.test.feature.pack.groupId: org.wildfly
        server.test.feature.pack.artifactId: wildfly-ee-galleon-pack
        server.version: 32.0.1.Final

    xp5, e.g. 8.0.2.GA-xp-5.0.0.GA-CR2:
		wildfly.channel.manifest.groupId: org.jboss.eap.channels
		wildfly.channel.manifest.artifactId: eap-xp-5.0
		server.version: 1.0.0.GA-redhat-00005

		server.test.feature.pack.groupId: org.jboss.eap.xp
        server.test.feature.pack.artifactId: wildfly-galleon-pack
        server.version: 5.0.0.GA-redhat-00005

    eap8, e.g. 8.0.2:
		wildfly.channel.manifest.groupId: org.jboss.eap.channels
		wildfly.channel.manifest.artifactId: eap-8.0
		server.version: 1.2.0.GA-redhat-00009

		server.test.feature.pack.groupId: org.jboss.eap
        server.test.feature.pack.artifactId: wildfly-ee-galleon-pack
        server.version: 8.0.3.GA-redhat-00001
```

I suppose we can use the `wildfly.channel.manifest.groupId` and `wildfly.channel.manifest.artifactId` but we need to be able to differentiate the version

Reply:
We could change the channel version to use a different property that defaults to `server.version`. That probably makes the most sense really. I'd rather not have multiple properties for group id's and artifact id's.

Reply:
now it should work with e.g.:

```bash
mvn -f testsuite/pom.xml \
	clean verify \
	-Dts.bootable=true -Ddefault=false \
	-Ddisable.microprofile.tests \
	-Dversion.org.wildfly.jar.plugin=9.0.1.Final-redhat-00009 \
	-Dserver.test.feature.pack.groupId=org.jboss.eap.xp \
	-Dserver.test.feature.pack.artifactId=wildfly-galleon-pack \
	-Dserver.version=5.0.0.GA-redhat-00005 \
	-Dts.bootable.channel \
	-Dwildfly.channel.manifest.groupId=org.jboss.eap.channels \
	-Dwildfly.channel.manifest.artifactId=eap-8.0-plus-eap-xp-5.0 \
	-Dwildfly.channel.manifest.version=1.0.0.GA-redhat-00005 \
	-Dmaven.repo.local=/tmp/jboss-eap-8.0.2.GA-xp-5.0.0.GA-CR2-patched-maven-repository/jboss-eap-8.0.2.GA-maven-repository/maven-repository \
	--batch-mode --fail-at-end -Denforcer.skip \
	-Dserver.home=/tmp/jboss-eap-8.0.2.GA-xp-5.0.0.GA-CR2-provisioned/jboss-eap-xp-5.0 \
	-Dversion.resteasy.testsuite=6.2.7.Final-redhat-00001 \
	-Dmaven.test.failure.ignore=true -Denforcer.skip
```